src/components/main-area/MainArea.scss

```diff
@@ -15,12 +16,14 @@ $sidebar-width: 20rem;
   left: -$sidebar-width;
   bottom: 0;
   width: $sidebar-width;
+  z-index: $zindex-fixed - 5;
   overflow-x: hidden;
   overflow-y: auto;
   background: $gray-200;
   box-shadow: 0 0 4px rgba(0, 0, 0, 0.25);
   transition: left 200ms ease;
   will-change: left;
+  overscroll-behavior: contain;
```

Comment:
I was removing all of these because I does not have any effect. This is testing artifact. Thank you for pointing 🙂

Reply:
I wanted to fix over scroll behavior in the following way:
- _toolbar_ and _sidebar_ should stay fixed at the top if user over scrolls
- the rest of content over scrolls with user action
- if user scrolls the sidebar, its content over scrolls with user action
- gray background covers whole area, i.e. if user over scrolls, no white lines should appear on the top and the bottom

In addition, body styles should change only with `componentDidMount` and `componentWillUnmount`

Reply:
> In addition, body styles should change only with componentDidMount and componentWillUnmount

Is there any specific reason for it?

Reply:
I mean it kinda does make sense for me. I just wanted to know whether this is considered as bad practice.

Reply:
Because the application is mounted by React, it may be unmounted and another one can be mounted afterwards.

I expected that the Reader may be kind of disabled in favour of another application that overlays the Reader. These styles are kind of styles I would like to disable. Otherwise this would not come into my mind.

Although, if React Helmet removes stuff when parent component is unmounted, there is no problem. We can put any `<link>` to CSS and it will be removed when necessary.

Reply:
Makes sense. Thanks

> Although, if React Helmet removes stuff when parent component is unmounted, there is no problem. We can put any <link> to CSS and it will be removed when necessary.

As fair as I know it doesn't.